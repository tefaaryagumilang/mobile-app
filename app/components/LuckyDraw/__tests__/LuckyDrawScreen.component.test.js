import React from 'react';
import renderer from 'react-test-renderer';

import LuckyDrawScreen from '../LuckyDrawScreen.component';

describe('LuckyDrawScreen component', () => {
  it('LuckyDrawScreen: renders correctly', () => {
    const tree = renderer.create(<LuckyDrawScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
