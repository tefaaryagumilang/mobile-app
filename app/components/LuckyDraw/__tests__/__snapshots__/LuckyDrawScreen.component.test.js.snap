// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LuckyDrawScreen component LuckyDrawScreen: renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#FFFFFF",
      "flex": 1,
    }
  }
>
  <RCTScrollView>
    <View>
      <View
        style={
          Object {
            "backgroundColor": "#FFFFFF",
            "flex": 1,
          }
        }
      >
        <KeyboardAwareScrollView
          extraHeight={120}
          keyboardShouldPersistTaps="handled"
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "flex": 1,
                "justifyContent": "center",
                "paddingBottom": 10,
                "paddingTop": 110,
              }
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "paddingTop": 20,
                  },
                  Object {
                    "color": "#000",
                    "fontSize": 12,
                  },
                  Object {
                    "color": "#666",
                  },
                ]
              }
            >
              Anda belum memiliki nomor Undian
            </Text>
          </View>
        </KeyboardAwareScrollView>
      </View>
    </View>
  </RCTScrollView>
</View>
`;
